src/eko/anomalous_dimensions/as1aem1.py

```diff
@@ -14,6 +14,35 @@
 from . import harmonics
 
 
+@nb.njit("u1(u1)", cache=True)
+def number_of_up_flavors(nf):
```

Comment:
Even `uplike_flavors` sounds not that bad...
(just to tell that `number_of` sounds a bit verbose and redundant, while `up` or `uplike` it's really a matter of preferences...)